component/attr_cache/attr_cache.go

```diff
@@ -172,34 +173,16 @@ func (ac *AttrCache) deleteDirectory(path string, time time.Time) {
 	// When we delete directory a, we only want to delete a/, a/b, and a/c.
```

Comment:
This comment block, starting with "For example..." occurs many times in the code. It has to do with prefix searches, which we don't do anymore. So all those comments (and likely many others) are outdated and can be removed.